### ISSUE TITLE
Separate AdminUI from `@keystone-6/core`

### ISSUE THREAD

**Issue by dcousens** [keystonejs/keystone#8916]

**Comment by mmachatschek** [comment]
this would be very much appreciated. opens a lot of ground that are currently only possible with workarounds (serverless admin ui)

**Comment by dcousens** [comment]
@mmachatschek indeed! This was actually meant to be a draft private issue 😅, but I'll add my proposal as a public feature request to gather some feedback, hopefully early next week 📆

**Comment by junaid33** [comment]
I started decoupling the Admin UI from Keystone 6 for my [Openfront](https://github.com/openshiporg/openfront) project since I wanted to replace @keystone-ui/* and emotion with shadcn/ui and Tailwind. I just finished recreating the dashboard using Tailwind and it's deployed alongside my full Next.js application as an app directory. Could be worth a look if you need a starting point.
